packages/dx-react-grid/docs/reference/create-row-cache.md

```diff
@@ -0,0 +1,30 @@
+# createRowCache
+
+A function that instantiates a default implementation of row cache. The cache is advised to be used in [lazy load mode](../guides/lazy-loading.md) to reduce the number of network requests.
+
+## User reference
+
+### Arguments
+
+Name | Type | Default | Description
+-----|------|---------|------------
+pageSize | number | 100 | The number of rows on virtual page. It must be equal to the value of pageSize property of the [VirtualTableState](virtual-table-state.md#properties) plugin.
+capacity | number | Infinity | The maximum number of pages that cache should store. When the reaches its capacity, it discards the least recently used virtual page.
+
+### Return value
+
+Type | Description
+-----|------------
+RowCache | A row cache.
```

Comment:
Can we add a reference to RowCache interface?
Can we add a section `How to import`?

Reply:
It's a good idea to add a usage example.
The RowCache interface described below, I think it's better to keep them in a single document

Reply:
No, I mean other points.

- Add [RowCache](`#RowCache`) link.
- Add import section as we did here
![image](https://user-images.githubusercontent.com/15093892/59030145-d13e2d00-8868-11e9-869b-f07ef0d3a12f.png)